docs/rules/object-shorthand.md

```diff
@@ -100,6 +100,37 @@ You can set the option in configuration like this:
 }
 ```
 
+While set to `"always"`, `"methods"`, or `"properties"`, shorthand syntax using string literal keys can be ignored using the optional parameter `"avoidQuotes"`. This will prevent warnings for properties with string keys, even if the name of the keys exactly match the name of the reference. It will also prevent warnings on longform syntax for methods with string keys, and will warn for shorthand method syntax where the key is a string literal. Note: The first parameter must be specified when using this optional parameter.
```

Comment:
"name of the reference" is unclear, I'd just remove the phrase starting at ", even if..."